public/v4/apps/affine.yml

```diff
@@ -0,0 +1,136 @@
+captainVersion: 4
+services:
+  #affine:
+  $$cap_appname:
+    image: ghcr.io/toeverything/affine-graphql:$$cap_affine_version
+    command:
+      ['sh', '-c', 'node ./scripts/self-host-predeploy && node ./dist/index.js']
+    ports:
+      - '3010:3010'
+      - '5555:5555'
```

Comment:
What is this used for? Can it be removed?

Reply:
I'm not sure why that port is used :/
https://github.com/toeverything/AFFiNE/discussions/5975#discussioncomment-10897370
I'll leave it from the time being